install/kubernetes/cilium/templates/hubble-ui/deployment.yaml

```diff
@@ -65,7 +65,7 @@ spec:
           value: "8090"
         {{- if .Values.hubble.relay.tls.server.enabled }}
         - name: FLOWS_API_ADDR
-          value: "hubble-relay:443"
+          value: "hubble-relay:{{ include "relay.servicePort" $ }}"
```

Comment:
We could extract the `FLOWS_API_ADDR` from the condition here correct?

Reply:
Good call.

Reply:
Sorry if I was unclear but I meant the `value:` part as well since it is the same in both branches. Then, the else block can be removed since it would be empty.

Reply:
Oh, gotcha. That makes a lot of sense. I just pushed this change.